scripts/functional.js

```diff
@@ -60,7 +60,13 @@ const testsDir = src || path.join(__dirname, "../test/functional");
       return true;
     })
     .browsers("chrome")
-    .run();
+    .run({
+      // When using React's ErrorBoundary, even if the boundary catches an error, it doesn't swallow the
+      // error. Because the error isn't swallowed, TestCafe would typically immediately make the test fail,
+      // making it difficult to have a test that passes after asserting that the ErrorBoundary displayed
+      // a proper message.
+      skipJsErrors: true
```

Comment:
This is an interesting situation. We're using our ErrorBoundary for displaying an error message not only for unforeseen JavaScript errors, but also for situations like when the user doesn't have appropriate AEP access. If we were _only_ using ErrorBoundary for unforeseen JavaScript errors (and handling foreseen errors in a different way), then we would probably leave `skipJsErrors` as `false`.

In our ErrorBoundary, we're also showing the message that comes from the error object. If the error is an unforeseen JavaScript error, we might not want to be showing that message to the user, since it's rather technical and the user can't do anything about it anyway. But when it's a foreseen error, like not having AEP access, the error object has a message that's user-friendly.

Anyway, this has me re-considering how we're dealing with error conditions, but I didn't really start thinking about it until this PR was mostly complete. I'd like to talk through it before changing it.

Reply:
Yeah, I agree. I'm going to see how some of this error management plays out in the other views and decide if/when we want to change how we're handling foreseen error situations.